static/docs/user-guide/experiments/branches.md

```diff
@@ -0,0 +1,119 @@
+# How to Manage Experiments by Branches
+
+You can use a different Git branch for each experiment.
+
+<p align="center">
+<img src="/static/img/user-guide/experiments/branches.png" />
+</p>
+
+This is usually more flexible than managing experiments by tags, since you can
+easily base a new experiment on any of the previous experiments.
+
+## Examples
+
+An example of managing experiments by branches can be seen on the
+[Deep Dive Tutorial](https://dvc.org/doc/tutorials/deep/reproducibility).
+
+These interactive tutorials also manage experiments by branches:
+
+- [Pipelines](https://katacoda.com/dvc/courses/tutorials/pipelines) - Using DVC
+  commands to build a simple ML pipeline.
+- [MNIST](https://katacoda.com/dvc/courses/tutorials/mnist) - Classify images of
+  hand-written digits using the MNIST dataset.
+
+## How it works
+
+### Commit and branch
+
+Let's say that we are working on the branch `master` and at the end of the
+experiment we want to save it on a branch named `unigrams`. We can do it like
+this:
+
+```dvc
+$ git commit -am 'Evaluate'
+$ dvc commit   # just to make sure all the data is committed
+$ git checkout -b unigrams
```

Comment:
why not `git branch unigrams`?

Reply:
Actually, `git branch` is pretty new and we don't use it in the docs. But we do use `git checkout` a lot. I would open a separate terminology-focused issue to decide whether to change all instances or not.

Reply:
Sorry I was confused about `git branch` being new lol. I was thinking of some new commands. But still we use `git checkout -b` a lot already so this seems consistent with existing docs, and is correct right?